ush/setup.sh

```diff
@@ -1285,9 +1285,11 @@ The CCPP physics suite specified in CCPP_PHYS_SUITE is not supported:
   check_var_valid_value \
     "CCPP_PHYS_SUITE" "valid_vals_CCPP_PHYS_SUITE" "${err_msg}"
 #
-# If running the RUN_FCST_TN task...
+# If running the MAKE_ICS_TN, MAKE_LBCS_TN, or RUN_FCST_TN task...
 #
-  if [ "${RUN_TASK_RUN_FCST}" = "TRUE" ]; then
+  if [ "${RUN_TASK_MAKE_ICS}" = "TRUE" ] || \
+     [ "${RUN_TASK_MAKE_LBCS}" = "TRUE" ] || \
+     [ "${RUN_TASK_RUN_FCST}" = "TRUE" ]; then
 #
```

Comment:
Why include MAKE_ICS and MAKE_LBCS in this "if" statement?

Reply:
Like above, this is a bug fix to ensure that the variables that reference the CCPP suite definition file are properly defined if running the `MAKE_ICS` or `MAKE_LBCS` task but not the `RUN_FCST` task.  Again, apparently a configuration that wasn't tested, or maybe this has already been fixed in `develop`.

Reply:
Got it!  Thanks!